src/utils/tests/frame.spec.js

```diff
@@ -72,17 +72,6 @@ describe("function names", () => {
 
       expect(formatDisplayName(frame)).toEqual("baz");
     });
-
```

Comment:
hmm i guess it would be difficult since the implementation is in css now... if you have any ideas on how to make that work, go for it, otherwise ignore this comment